lib/index.js

```diff
@@ -29,16 +29,24 @@ internals.Client = function () {
         http: new Http.Agent({ maxSockets: Infinity }),
         httpsAllowUnauthorized: new Https.Agent({ maxSockets: Infinity, rejectUnauthorized: false })
     };
+
+    this._defaults = defaults || {};
 };
 
 Hoek.inherits(internals.Client, Events.EventEmitter);
 
+internals.Client.prototype.defaults = function (options) {
+
+    options = Hoek.applyToDefaultsWithShallow(options, this._defaults, ['agent', 'payload', 'downstreamRes']);
```

Comment:
this will fail lint in the next lab.

Make this 
```
var settings = Hoek.applyToDefaultsWithShallow(options, this._defaults, ['agent', 'payload', 'downstreamRes']);

return new internals.Client(settings);
```

to avoid variable shadowing